programs/gamma/src/instructions/swap_base_output.rs

```diff
@@ -262,7 +262,8 @@ pub fn swap_base_output<'c, 'info>(
         },
         input_transfer_fee,
         output_transfer_fee,
-        base_input: false
+        base_input: false,
+        dynamic_fee: result.dynamic_fee,
```

Comment:
this is the fee charged, does it also make sense to emit the actual rate? I think we can calculate that but thinking if that can reduce the code in backend service.